styles/components/User-Widget.scss

```diff
@@ -0,0 +1,66 @@
+@import "../settings";
+.User-Widget {
```

Comment:
UserWidget.scss

.UserWidget {}
.UserWidget-icon {}
.UserWidget-balance {}

We use this naming convention https://github.com/gilbox/css-bliss#naming